lock-api/src/main/java/com/palantir/lock/client/LockLeaseService.java

```diff
@@ -42,7 +42,7 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 
-class LockLeaseService implements AutoCloseable {
+public class LockLeaseService implements AutoCloseable {
```

Comment:
We have to be careful - if I'm not mistaken, making this public exposes more API, and also possibly exposes us to ABI breaks if the user should ever decide to do something naughty. It's possibly not an issue, but there's a reason why we had to do a _lot_ of wrangling w.r.t. the lock watch manager. Let's sync up about this offline.